### PR TITLE
New package: remhind-0.1.1

### PR DESCRIPTION
**Pull request by Eluminae**
In response of https://github.com/void-linux/void-packages/issues/18172

### CODE REVIEW

**Comment by Eluminae** [comment]
```
Traceback (most recent call last):
  File "/bin/remhind", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3251, in <module>
    def _initialize_master_working_set():
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3234, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3263, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'xdg' distribution was not found and is required by remhind
```
with python3-xdg and python2-xdg

**Comment by Eluminae** [comment]
@CameronNemo  Here is a working dep tree using a new "python3-xdg2" package. How should I name it ?

File: srcpkgs/python3-aionotify/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-aionotify'
+pkgname=python3-aionotify
+version=0.2.0
+revision=1
+archs=noarch
+wrksrc="aionotify-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
```
Comment (xtraeme) [inline srcpkgs/python3-aionotify/template]: missing depends

Comment (Eluminae) [reply srcpkgs/python3-aionotify/template]: Which ones ?

Comment (Chocimier) [reply srcpkgs/python3-aionotify/template]: Check yourself, e.g. by importing in clean masterdir.

File: srcpkgs/python3-xdg2/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-xdg2'
+pkgname=python3-xdg2
```
Comment (Chocimier) [inline srcpkgs/python3-xdg2/template]: python3-xdg-variables maybe? pyxdg do way more.

File: srcpkgs/python3-aionotify/template (hunk 1 above)
Comment (Eluminae) [reply srcpkgs/python3-aionotify/template]: Yup, and it just works...

File: srcpkgs/python3-xdg2/template (hunk 2 above)
Comment (Eluminae) [reply srcpkgs/python3-xdg2/template]: Seems a decent name. Renamned

File: srcpkgs/python3-aionotify/template (hunk 1 above)
Comment (CameronNemo) [reply srcpkgs/python3-aionotify/template]: I checked the upstream stated `requirements.txt` and I do not see what is missing here...

Comment (Chocimier) [reply srcpkgs/python3-aionotify/template]: `depends="python3"` is needed.

**Comment by Eluminae** [comment]
> `depends="python3"` is needed.

Okay, true. I guessed python3-module build styles add this but no.

**Comment by Chocimier** [comment]
Some dependency missing for remhind?

```
Traceback (most recent call last):
  File "/usr/bin/remhind", line 11, in <module>
    load_entry_point('remhind==0.1.1', 'console_scripts', 'remhind')()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2443, in load
    return self.resolve()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.8/site-packages/remhind/__main__.py", line 11, in <module>
    from .events import check_events, CalendarStore
  File "/usr/lib/python3.8/site-packages/remhind/events.py", line 15, in <module>
    gi.require_version('Notify', '0.7')
  File "/usr/lib/python3.8/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace Notify not available
```

**Comment by Eluminae** [comment]
> Some dependency missing for remhind?

It seems yes. Fixed adding aionotify and notify as deps. Dunno how I
missed them.

**Pull request merged**